lib/reportportal-agent.js

```diff
@@ -9,7 +9,7 @@ class ReportportalAgent {
             project: '',
         }, conf);
         this.client = new ReportportalClient(config);
-        this.launchInstance = conf.id ? (this.client.startLaunch({id:conf.id, tags: conf.tags})): (this.client.startLaunch({tags: conf.tags}));
+        this.launchInstance = conf.id ? (this.client.startLaunch({id:conf.id, attributes: conf.attributes, description: conf.description})): (this.client.startLaunch({attributes: conf.attributes, description: conf.description}));
```

Comment:
The difference in the cases just in existed `id`. I suggest to define launch object before and reuse it in `this.client.startLaunch` function.